analyzere/__init__.py

```diff
@@ -20,6 +20,9 @@
 upload_chunk_size = 16 * one_megabyte
 tls_verify = True
 user_agent = 'analyzere-python 0.8-dev'
+connection_pool_maxsize = 10
+retry_strategy_total = 25
+retry_strategy_backoff_factor = 0.1
```

Comment:
any special reasons for any of these default values?

am I right in understanding: if unchanged, any request that fails will be retried up to 25 times before an exception is thrown in the client code?

Reply:
Yeah, good question, this is just what was in the ticket but the default for requests is no retries, so I'll update it to no retries to preserve existing behaviour